app/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "node_modules/react-native/packager/packager.sh --root js"
+    "start": "node_modules/react-native/packager/packager.sh --root js",
+    "start-windows": "node_modules\\react-native\\packager\\packager.sh --root js"
```

Comment:
From Windows 10, need to use backward slash in the file path, which is inconsistent to the Unix system. So, in order to still share this file across platforms, use separate script for Windows. And the command to start packager need to be changed to: npm run-script start-windows for Windows platform, and npm run-script start for other platform